MdeModulePkg/Library/UefiBootManagerLib/UefiBootManagerLib.inf

```diff
@@ -120,3 +122,4 @@
   gEfiMdeModulePkgTokenSpaceGuid.PcdBootManagerMenuFile                     ## CONSUMES
   gEfiMdeModulePkgTokenSpaceGuid.PcdDriverHealthConfigureForm               ## SOMETIMES_CONSUMES
   gEfiMdeModulePkgTokenSpaceGuid.PcdMaxRepairCount                          ## CONSUMES
+  gEfiMdeModulePkgTokenSpaceGuid.PcdBootManagerInBootOrder                  ## CONSUMES   ## MSCHANGE

```

Comment:
nit: swap to MU_CHANGE

Reply:
Removed entirely, this is from a different commit that got lumped in with this one sometime in the past.